matRad/gui/widgets/matRad_ViewingWidget.m

```diff
@@ -514,7 +514,7 @@ function notifyPlotUpdated(obj)
                     end
                     this.updateValues();
                     %doUpdate = this.checkUpdateNecessary({'pln_display','ct','cst','resultGUI','image_display'},evt);
-                    if  this.checkUpdateNecessary({'resultGUI','image_display'},evt)
+                    if  this.checkUpdateNecessary({'resultGUI','image_display','cst_param'},evt)
```

Comment:
What I wonder is if we really need to recreate the IsoDoseLine-Cache? Shouldn't the visibility be used somewhere else in the Widget for turning the VOI contours on/off?